workers/unity/Packages/com.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -215,6 +231,13 @@ private static void CopySchemaFiles(string schemaRoot, KeyValuePair<string, stri
             }
         }
 
+        private static Dictionary<string, string> GetPackagePaths(this Dictionary<string, string> packages)
```

Comment:
Maybe `GetLocalPathsInPackage` ? To make it explicit this looks for local dependencies and resolves the path? 

Also don't like that this is an extension method

Reply:
Extension methods = magic and should be used carefully 😉